Gemfile

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 ruby '2.4.1'
 
 git_source(:github) do |repo_name|
-  repo_name = '#{repo_name}/#{repo_name}' unless repo_name.include?('/')
-  'https://github.com/#{repo_name}.git'
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
+  "https://github.com/#{repo_name}.git"
 end
```

Comment:
@sinpai isn't `bundle install` works good without?
```ruby
git_source(:github) do |repo_name|
  ...
end
```

I mean can we just remove it?)

Reply:
Ok 👍